src/forms/labels/input-label.js

```diff
@@ -106,7 +106,8 @@ function InputLabel({
         </label>
       )}
       {tooltip && (
-        <span
+        <button
+          type="button"
```

Comment:
Ideally, I'd like to remove this altogether since there are much better tooltip implementations out there but I'm considering that out of scope for this change